### PR TITLE
Peer Instruction XBlock for course teams

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2285](https://openedx.atlassian.net/browse/DOC-2285)

The PR adds documentation about the UBC-created peer instruction XBlock for course teams. I provide information about how online delivery of this type of exercise is different from in-classroom, what teams do to prepare an assignment, and then how they enable and use the PI component in Studio to add this type of exercise to their courses. 
As background on how widespread these question types are in classrooms, see the [Turn to Your Neighbor](http://blog.peerinstruction.net/2015/01/08/3-ways-to-write-effective-questions/) blog. 
### Date Needed

This feature is in use on Edge and I would like to get all documentation ready for partners and open source users by 15 Jan. 
### Reviewers
- [x] Subject matter expert: @xcompass 
- [ ] Doc team review (dev edit): @catong @pdesjardins @srpearce 
- [x] Product review: @explorerleslie 
  FYI: @mmacfarlane @jbarciauskas @cptvitamin @bradenmacdonald ; please tag others as appropriate.
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Sandbox until 16 January 2016 only!

log in as staff@example.com
- [ ] Studio: https://studio-ubcpi.sandbox.edx.org/container/block-v1:edX+CSM101x+Dec2015+type@vertical+block@df2394ecdcc44443b87a2148b4cc6b23
- [ ] LMS problem with histogram: https://ubcpi.sandbox.edx.org/courses/course-v1:edX+CSM101x+Dec2015/courseware/801f910433b64e62bc3246f69da0ff4f/aa69e403c53e4224aef6d8ef3f3d7458/3?activate_block_id=block-v1%3AedX%2BCSM101x%2BDec2015%2Btype%40vertical%2Bblock%40df2394ecdcc44443b87a2148b4cc6b23
### Post-review
- [x] Add description to release notes task as a comment
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/exercises_tools/peer_instruction_tool.rst [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+#. A final response, which also includes an answer choice and revised
+   explanation.
+
+Learners also receive an explanation for the correct answer choice. After 10
+learners complete the assignment, class breakdown histograms show the
+percentage of responding learners who selected each of the answer choices, both
+initially and after reviewing peer responses.
+
+.. note:: While Peer Instruction assignments include multiple choice
+  questions, these questions cannot be graded in the edX LMS.
```
Comment (xcompass) [inline en_us/shared/exercises_tools/peer_instruction_tool.rst]: The participation grade is included in the current PR.

File: en_us/shared/exercises_tools/peer_instruction_tool.rst [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+   * If your question has more than three answer choices, you select **Add New
+     Example**, and then identify the answer choice and enter the example
+     answer for each of the other answer choices.
+
+   * To delete an example answer, select the **X** next to the list of numbered
+     choices.
+
+#. Select **Save**. The system performs an edit check to make sure that the
+   component has an example answer for each of the answer choices.
+
```
Comment (xcompass) [inline en_us/shared/exercises_tools/peer_instruction_tool.rst:242]: Maybe add: If **Save** is not available, there is one or more invalid input in the form. Please check the exclamation mark icon beside the button for more details.

**Comment by xcompass** [comment]
Great work. Thanks @lamagnifica :+1:

**Comment by explorerleslie** [comment]
besides Pan's comments, :+1:

File: en_us/shared/exercises_tools/peer_instruction_tool.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/exercises_tools/peer_instruction_tool.rst]: sweet!

**Comment by lamagnifica** [comment]
@bradenmacdonald FYI. Note sandbox in description.

**Comment by bradenmacdonald** [comment]
Thanks @lamagnifica :)

**Pull request merged**